apisix/cli/ngx_tpl.lua

```diff
@@ -100,18 +100,14 @@ http {
     }
 
     server {
-        {% if use_apisix_base then %}
```

Comment:
Now, this API can be run in a normal worker

Reply:
The export API will no longer be exposed to privileged processes, which provides isolation of HTTP traffic from root privileges for enhanced security.
Therefore this is no longer needed.